### PR TITLE
fix: Autoteleport does incorrect calculations

### LINKED ISSUE #11059: Autoteleport does incorrect calculations
### Context

For testing I was minting some NFTs and I found out that app is doing incorrect calculations.

Test case. 

1. I had some DOT (less than 0.27 DOT) and the minting price was 3 DOT
2. [App decided to teleport 2.73 DOT](https://assethub-polkadot.subscan.io/xcm_message/polkadot-754cc374ad7364bc1aed254d1f2a8231b638ccb0)
3. So in result I had 3 DOT (bear in mind that I need ED and tx fee)
4. Minting failed

### Fix 

Ensure that enough DOT is teleported :) (I need at least  3.05 DOT in total to have enough)

### SCR 

Screenshot 2024-09-30 at 12 04 51

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

@JustLuuuu @exezbcz @kodadot/internal-dev  need help testing , let me know if it still doesn't work 

Did some testing , will continue doing more tomorrow

- [x] add item deposit to drop mint calculation
- [x] add autotelport action fees 
- [x] Closes #11059

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-10-01 at 18 07 55@2x

CleanShot 2024-10-01 at 18 07 36@2x

### CODE REVIEW

**Comment by hassnian** [comment]
did more testing and it's working fine


https://github.com/user-attachments/assets/fed789d7-060b-4f6d-ae8b-7db271ad2178

File: components/drops/useDrops.ts [hunk 1]
```diff
@@ -76,9 +78,7 @@ export const useDropMinimumFunds = (amount = ref(1)) => {
     = useMultipleBalance()
 
   const price = computed<number>(() => Number(drop.value?.price) || 0)
-  const minimumFunds = computed<number>(() =>
-    price.value ? amount.value * price.value : 0,
-  )
+  const minimumFunds = computed<number>(() => amount.value * (price.value + itemDeposit.value))
```
Comment (vikiival) [inline components/drops/useDrops.ts:81]: + ed + oneTXFee

**Review by vikiival — COMMENTED** [review]
~

File: components/drops/useDrops.ts (hunk 1 above)
Comment (hassnian) [reply components/drops/useDrops.ts:81]: Auto teleport handles that

**Pull request merged**